### ISSUE TITLE
dump command must export keys prefixed with gluu/config

### ISSUE THREAD

**Issue by iromli** [GluuFederation/docker-config-init#6]
There's problem when Consul KV has multiple directories. The `dump` command must not export keys that aren't stored under `gluu/config` directory.

**Comment by iromli** [comment]
An example of config stored in Consul KV:

- `gluu/config/hostname`

Proposed format in JSON file:

```
{
    "_config": {
        "hostname": "example.com"
    }
}
```

Given `_config` key as seen above, we can avoid importing arbitrary keys in `load` command (see https://github.com/GluuFederation/docker-config-init/issues/7).

**Comment by iromli** [comment]
Fixed in 577c4bc1ca161f5dde53eeabfd77f6f67cc60f70
